pkg/internal/database/logger_adapter.go

```diff
@@ -0,0 +1,49 @@
+package database
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"gorm.io/gorm/logger"
+)
+
+// SlogGormLogger implements gorm/logger.Interface using slog
+type SlogGormLogger struct {
+	logger *slog.Logger
+}
+
+// Info logs info-level messages
+func (l *SlogGormLogger) Info(_ context.Context, msg string, args ...any) {
+	l.logger.Info(fmt.Sprintf(msg, args...))
```

Comment:
oh, maybe we should use methods `XyzContext` and pass the context as it is there already in arguments